src/ParseCallMap.hs

```diff
@@ -0,0 +1,313 @@
+{-# language FlexibleInstances, OverloadedStrings, ViewPatterns, GeneralizedNewtypeDeriving #-}
+module ParseCallMap (fromFile, fromSource, CallMapParseError) where
+
+import Prelude hiding (exp)
```

Comment:
ghc complained - although it looks like that was a previous iteration of this code that defined and `exp` expression parser which eventually became `theForm`